android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/navigation/NavigationMenuConfig.kt

```diff
@@ -24,7 +26,15 @@ data class NavigationMenuConfig(
   val id: String,
   val visible: Boolean = true,
   val icon: String? = null,
+  val menuIconConfig: MenuIconConfig? = null,
```

Comment:
I think the property `icon` is a duplicate of `menuIconConfig`

Reply:
Will remove the old one